singularity-opac/photons/non_cgs_photons.hpp

```diff
@@ -246,7 +246,8 @@ class MeanNonCGSUnits {
                   const Real temp_unit)
       : mean_opac_(std::forward<MeanOpac>(mean_opac)), time_unit_(time_unit),
         mass_unit_(mass_unit), length_unit_(length_unit), temp_unit_(temp_unit),
-        rho_unit_(mass_unit_ / (length_unit_ * length_unit_ * length_unit_)) {}
+        rho_unit_(mass_unit_ / (length_unit_ * length_unit_ * length_unit_)),
+        inv_emiss_unit_(length_unit_ * time_unit_ * time_unit_ / mass_unit_) {}
```

Comment:
Is this not 
```suggestion
        inv_emiss_unit_(length_unit_ * time_unit_ * time_unit_ * time_unit_ / mass_unit_) {}
```
? since
`<J> = <dE / d^3x dt> = g cm^2 / (s^2 cm^3 s) = g / (cm s^3)`

Reply:
Oh I see you add the extra `time_unit_` in separately later... but any reason to not just include that here?

Reply:
This may've been to conform to the pattern for the non-mean variant - will double-check.

Reply:
Just made this change - thanks!

Reply:
🙌